RibonDAO/interface
# refactor: update contribution events

# Description

- Update events in contribution flow

## Type of change

Please delete options that are not relevant.


## How to test

use the `yarn start-debug` command and check if the events are being logged correctly
